media-libs/shaderc/files/shaderc-2017.2-disable-git-versioning.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index b9bd0df..9284273 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -58,9 +58,3 @@ add_subdirectory(libshaderc_util)
+ add_subdirectory(libshaderc)
+ add_subdirectory(glslc)
+ add_subdirectory(examples)
+-
+-add_custom_target(build-version
```

Comment:
Extra question: do you *really* need to strip it when you remove the dep below? FWICS it doesn't use `ALL`, so it should be just a no-op.

Reply:
Indeed. Will check.

Reply:
Yes, wasn't needed.